src/qililab/__init__.py

```diff
@@ -27,3 +30,9 @@
 from .utils import Loop, Wait
 from .utils.load_data import load
 from .waveforms import *
+
+with contextlib.suppress(NameError, ImportError):
+    # Since Ipython magic methods can only be imported from inside a Jupyter Notebook,
+    # here we first check that `get_ipython` exists (which means we are inside a Jupyter Notebook)
+    get_ipython()  # type: ignore  # noqa: F405 # pylint: disable=undefined-variable
+    from .slurm import submit_job
```

Comment:
Weird, this line is not covered... This means that the function `submit_job` is never imported! How are the tests passing then? 🤔

Reply:
my guess this is because I was doing `import qililab.slurm` instead of `import qililab` because submit_job was not found in the latter case. finally I do `session_ip.run_cell(raw_cell="import qililab.slurm\nimport qililab")` to cove both cases